docs/pages/versions/v52.0.0/sdk/notifications.mdx

```diff
@@ -471,7 +491,11 @@ For Android:
 
 For iOS:
 
-- In the array under the `ios.infoPlist.UIBackgroundModes` key in your [app config](/workflow/configuration/), add the values `remote-notification` and `processing`.
+- background mode must be specified in **Info.plist** file. See [Background notifications
+  configuration](#background-notification-configuration).
+- You must use a [development
+  build](/develop/development-builds/introduction/) to use background notifications since it is not
+  supported in the Expo Go app.
```

Comment:
```suggestion
- You must use a [development build](/develop/development-builds/introduction/) to use background notifications since it is not supported in the Expo Go app.
```